frontend/src/metabase/visualizations/types/visualization.ts

```diff
@@ -236,8 +236,7 @@ export type VisualizationDefinition = {
   placeHolderSeries?: Series;
 
   transformSeries?: (series: Series) => TransformedSeries;
-  // TODO: remove dependency on metabase-lib
-  isSensible: (data: DatasetData, query?: Query) => boolean;
+  isSensible: (data: DatasetData) => boolean;
```

Comment:
I assume the "dependency on metabase-lib" TODO here refers to the `Query` type and `databaseSupportsPivotTables`, which we just removed.